frontend/kesaseteli/employer/src/components/header/__tests__/Header.test.tsx

```diff
@@ -37,9 +37,6 @@ describe('frontend/kesaseteli/employer/src/components/header/Header.tsx', () =>
     await waitFor(() =>
```

Comment:
this seems to be false positive check, because query data is no longer cleared, it should have been failed. Something is wrong with the test.